Project.toml

```diff
@@ -22,6 +22,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Rotations = "6038ab10-8711-5258-84ad-4b1120ba62dc"
 Scratch = "6c6a2e73-6563-6170-7368-637461726353"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
We can remove `StaticArrays`, now that we have the `Core` version.

Reply:
We still need `StaticArrays` for `setindex` that we use in the `rotate` functions for the random walks